build/common.sh

```diff
@@ -322,6 +322,10 @@ function kube::build::destroy_container() {
   "${DOCKER[@]}" rm -f -v "$1" >/dev/null 2>&1 || true
 }
 
+function kube::build::docker_rootless() {
```

Comment:
```suggestion
function kube::build::is_docker_rootless() {
```

as it's a true/false return function

Reply:
done